### ISSUE TITLE
请教Windows Server下如何配置？

### ISSUE THREAD

**Issue by gazerhuang** [pengzhile/atlassian-agent#28]
救救运维小白，我开了台WinServer2012虚拟机，全新安装的atlassian-jira-software-8.1.0-x64，没有试用过。

直接下的release包，并放到C:\Jira\下。

在C:\Program Files\Atlassian\JIRA\bin\setenv.bat中修改了：
set JAVA_OPTS=%JAVA_OPTS% -Xms%JVM_MINIMUM_MEMORY% -Xmx%JVM_MAXIMUM_MEMORY% %JVM_CODE_CACHE_ARGS% %JVM_REQUIRED_ARGS% %DISABLE_NOTIFICATIONS% %JVM_SUPPORT_RECOMMENDED_ARGS% %JVM_EXTRA_ARGS% %JIRA_HOME_MINUSD% %START_JIRA_JAVA_OPTS% -javaagent:C:\Jira\atlassian-agent.jar 

之后运行服务，查看C:\Program Files\Atlassian\JIRA\logs\catalina.2019-04-30.log，并没有看到== agent working ==
试了下生成激活码填进去自然也不能用
哪里不对，请指点一下，谢谢。

**Comment by ShelMax** [comment]
Hi, I was have the same problem on Windows Server 2012, and resolved with this tutorial -> https://confluence.atlassian.com/adminjiraserver070/increasing-jira-application-memory-749383419.html
Use "Windows service" part and put in "Java Options:" this line -> -javaagent:C:\Jira\atlassian-agent.jar

p.s. On Windows in tomcat 8.5 ignored settings in "setenv.bat", I don't know why, but I was spend 1 day to find this solution, i hope it's will help you.

p.s.s. You will not see in logs "== agent working ==" but you will see agent was started, for example: "03-May-2019 05:04:13.296 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -javaagent:C:\Jira\atlassian-agent.jar"

**Comment by pengzhile** [comment]
> Hi, I was have the same problem on Windows Server 2012, and resolved with this tutorial -> https://confluence.atlassian.com/adminjiraserver070/increasing-jira-application-memory-749383419.html
> Use "Windows service" part and put in "Java Options:" this line -> -javaagent:C:\Jira\atlassian-agent.jar
> 
> p.s. On Windows in tomcat 8.5 ignored settings in "setenv.bat", I don't know why, but I was spend 1 day to find this solution, i hope it's will help you.
> 
> p.s.s. You will not see in logs "== agent working ==" but you will see agent was started, for example: "03-May-2019 05:04:13.296 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -javaagent:C:\Jira\atlassian-agent.jar"

do not use "Windows service"

**Comment by gazerhuang** [comment]
> Hi, I was have the same problem on Windows Server 2012, and resolved with this tutorial -> https://confluence.atlassian.com/adminjiraserver070/increasing-jira-application-memory-749383419.html
> Use "Windows service" part and put in "Java Options:" this line -> -javaagent:C:\Jira\atlassian-agent.jar
> 
> p.s. On Windows in tomcat 8.5 ignored settings in "setenv.bat", I don't know why, but I was spend 1 day to find this solution, i hope it's will help you.
> 
> p.s.s. You will not see in logs "== agent working ==" but you will see agent was started, for example: "03-May-2019 05:04:13.296 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -javaagent:C:\Jira\atlassian-agent.jar"

It works！Thank you very mach！But the same way can't be used for Confluence……let me try again.

**Comment by gazerhuang** [comment]
这个方法对于全套应该都有用，通过“程序名 //ES//服务名” 直接把javaagent加进去，剩下的就按教程走，Win上直接修改文件没有用的朋友可以试试。
再次谢谢@ShelMax ，Thank you.
